### PR TITLE
Set up github action for filing issue reports

### PR DESCRIPTION
**Pull request by dontcallmedom**
Only dispatch for now

### CODE REVIEW

File: .github/workflows/file-draf-issues.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+name: Submit draft issue reports from Strudy analysis
+jobs:
+  update:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Setup node.js
+      uses: actions/setup-node@v2
+      with:
+        node-version: 14
```
Comment (tidoust) [inline .github/workflows/file-draf-issues.yml:12]: Long term support version of Node is currently 16, any reason to stick to 14 for this new job?

```suggestion
        node-version: 16
```

**Review by tidoust — CHANGES_REQUESTED** [review]
Workflow filename should be `file-draft-issues.yml` (final `t` is missing in `draft`).

Why call it "draft" actually? Because it's only triggered by dispatch for now?

**Comment by dontcallmedom** [comment]
I called it draft (or meant to) because it doesn't file issues in the target repo, it just submits them as pull request for review. Should I just fix the typo or do you have another name to suggest?

**Review by tidoust — APPROVED** [review]
Naming is hard ;)

Since this job is really just a wrapper around the reporting script, perhaps reusing `file-issue-for-review` would be good. At least, that would make the association between the job and the script file more direct. Unless you think that association will diverge at some point, i.e. because the job will do more than setting things up and calling the script.

Anyway, up to you :)

**Pull request merged**